_content/_docs_signals/how_it_works.md

```diff
@@ -47,32 +47,34 @@ You can use Signals to
 * send out an email and/or Slack notification
 * send an addition escalation email if the issue persist for more than 1 hour
 
-## Components of a Signals watch
+## Building blocks of a Signals watch
 
-A Signals watch consists of the following components:
+The basic working principle of a watch goes as follows:
```

Comment:
I reworked this because it is not really possible create seperate concepts for checks and inputs, etc, IMHO